api/app/settings/common.py

```diff
@@ -870,3 +870,9 @@
 
 # Used for serializing and deserializing GenericForeignKey(used in metadata) using the natural key of the object
 SERIALIZATION_MODULES = {"json": "import_export.json_serializers_with_metadata_support"}
+
+# Controls the app domain used in emails (currently invites and change requests).
+# If set, domain stored with `django.contrib.sites` is disregarded.
+DOMAIN_OVERRIDE = env.str("FLAGSMITH_DOMAIN", "")
+# Used when no Django site is specified.
+DEFAULT_DOMAIN = "app.flagsmith.com"
```

Comment:
Was there a reason to use `app.flagsmith.com` here vs just `flagsmith.com` ?

Reply:
It was the default argument value – see [here](https://github.com/Flagsmith/flagsmith/blob/e009e58e0ef4bbc9d645b8b12098abafb7b41178/api/core/helpers.py#L5).

Do the cloud invites/workflows point to `flagsmith.com` instead? Is there a working codepath for this setting at all?

Reply:
>Do the cloud invites/workflows point to flagsmith.com instead? 

Yes, because the domain is set in the default Site object in our SaaS deployment. 

I actually don't think that default will ever be used unless you explicitly delete the default Site object that is created when you run the migrations.

Reply:
What’s the use case for overriding the domain with having a Site?

Do you think we should rather allow the users to define a fallback value and recommend them to clear the Sites/the SITE_ID setting?

Reply:
Resolved this in DMs with @matthewelwell.

For the time being we want to keep the config options as flexible as possible. Further down the road we might want to deprecate django_sites usage, but this will require us to revisit the [installation flow](https://docs.flagsmith.com/deployment/hosting/locally-api#environments-with-no-direct-console-access-eg-heroku-ecs) for some environments.

The docs have been updated to reflect the change ([PR](https://github.com/Flagsmith/flagsmith-docs/pull/340)).